operations/configuration_management/NCN_Node_Personalization.md

```diff
@@ -1,25 +1,62 @@
 # NCN Node Personalization
 
-When performing an upgrade or fresh install, NCN node personalization must be performed on the NCN management nodes to ensure the appropriate CFS layers are applied post-boot.
-This step involves configuring CFS to use the default `sat bootprep` files from the `hpc-csm-software-recipe` repository and applying the resulting configuration to the NCN management nodes.
-The same CFS configuration is used for post-boot personalization of master, storage, and worker NCNs. However, some individual parts of that configuration will only be applied to
-appropriate node types.
+- [Background](#background)
+- [Prerequisites](#prerequisites)
+- [Procedure](#procedure)
+  1. [Preparation](#1-preparation)
+  1. [Edit CPE and Analytics layers](#2-edit-cpe-and-analytics-layers)
+  1. [Disable CFS on management NCNs](#3-disable-cfs-on-management-ncns)
+  1. [Update CFS configuration and components](#4-update-cfs-configuration-and-components)
+  1. [Cleanup](#5-cleanup)
 
-The definition of the CFS configuration used for node personalization is provided in the `hpc-csm-software-recipe` repository in VCS.
-The following procedure describes how to correctly edit the `sat bootprep` files to be able to use them to perform node personalization.
+## Background
 
-1. (`ncn-m#`) Perform the steps in the [Accessing `sat bootprep` Files](Accessing_Sat_Bootprep_Files.md) procedure to gather a copy of the `sat bootprep` files.
+NCN node personalization refers to the process of CFS applying a configuration to a management NCN after it is booted.
+During CSM installs and upgrades, this CFS configuration must be created and set as the desired configuration in CFS for all management NCNs.
+The purpose is to ensure that the appropriate CFS layers are applied to the management NCNs after they boot.
 
-1. (`ncn-m#`) Create a local copy of the `management-bootprep.yaml` file and delete the `ncn-image-customization` configuration. The `ncn-personalization` configuration should be the only entry remaining in the file if completed correctly.
+The same CFS configuration is used for post-boot personalization of master, storage, and worker NCNs. However, some individual parts of that
```

Comment:
is there a reason for this line break between line 18 and 19?

Reply:
Line length. Like HTML, when the markdown is rendered, the line break goes away. So our source files usually start new lines somewhere after 60-80-ish characters.